### DISCUSSION TITLE
3 zone ride configuration

### DISCUSSION THREAD

**Discussion by ojungleboy** [corrados/edrumulus#53] [Q&A] [answered]
Hi there, thanks once again for all your work guys.

I am wondering if it is possible to configure a 3 zone ride. As far as I see, only dual zone triggers are available. Am I wrong?
Thank you

**Comment by corrados** [comment, marked as answer]
That is correct. Edrumulus only supports dual zone triggers.

I don't know how your 3 zone ride pad works. But maybe it is possible to use a separate trigger input for the third sensor. To make sure only one sound is played, you could try to use the crosstalk cancellation feature.

**Comment by ignotus666** [comment]
"Yamaha-style" 3 zone cymbals only use one stereo input, as opposed to Roland 3 zone cymbals, which use two. They are piezo/switch/switch pads and both switches are connected to ring, but one of them goes through a 10k resistor to enable the module to distinguish between them. It would be cool if these types of pads could be supported.

**Reply by ojungleboy** [reply]
Definetly! 10k resistor saves us some inputs in certain cases!

I am studying coding so I can try to contribute to the project and propose this changes. But it will take a while until.

Will keep looking this project for updates.

Thanks again!
